File: src/test/java/com/codeborne/json/assertions/JsonAssertions.java
package com.codeborne.json.assertions;

import java.io.InputStreamReader;
import java.io.Reader;

import static java.nio.charset.StandardCharsets.UTF_8;

public class JsonAssertions {
  public static JsonAssert assertThat(Object parsedJson) {
    return new JsonAssert(parsedJson);
  }

  static Reader file(String fileName) {
    return new InputStreamReader(JsonAssertions.class.getResourceAsStream('/' + fileName), UTF_8);
  }
}

File: src/test/java/com/codeborne/json/assertions/JsonAssertions.kt
package com.codeborne.json.assertions

import java.io.InputStreamReader
import java.io.Reader
import java.nio.charset.StandardCharsets

object JsonAssertions {
    fun assertThat(parsedJson: Any?): JsonAssert {
        return JsonAssert(parsedJson)
    }

    fun file(fileName: String): Reader {
        return InputStreamReader(JsonAssertions::class.java.getResourceAsStream("/$fileName"), StandardCharsets.UTF_8)
    }
}
